content/sources.adoc

```diff
@@ -6,6 +6,6 @@ include::sources/settings.adoc[leveloffset=+1]
 
 include::sources/workflow.adoc[leveloffset=+1]
 
-include::sources/cheatsheet.adoc[leveloffset=+1]
+include::sources/contributions.adoc[leveloffset=+1]
 
-include::sources/contributions.adoc[leveloffset=+1]
\ No newline at end of file
+include::sources/best_practices.adoc[leveloffset=+1]
```

Comment:
this line seems not working :
![image](https://user-images.githubusercontent.com/807839/132302048-2d9f11b2-66f0-4f39-9015-e89a87c52020.png)

Reply:
Yep, will delete it